rpc/rpcserver/log.go

```diff
@@ -23,6 +23,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
+var log = slog.Disabled
```

Comment:
why is this added?  the service implementations don't log anything at all currently.  this doesn't even add a way to modify the logger to be anything but the disabled logger.